src/coreclr/src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1249,6 +1253,40 @@ void CodeGen::genBaseIntrinsic(GenTreeHWIntrinsic* node)
     genProduceReg(node);
 }
 
+//------------------------------------------------------------------------
+// genX86BaseIntrinsic: Generates the code for an X86 base hardware intrinsic node
+//
+// Arguments:
+//    node - The hardware intrinsic node
+//
+void CodeGen::genX86BaseIntrinsic(GenTreeHWIntrinsic* node)
```

Comment:
Not needed for this PR, but we allowed `HW_Category_Scalar` intrinsics to be table driven in importation.
It would be nice to do the same here in codegen...